spec/services/comments/create_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ::Comments::Create do
+  subject { described_class.call(book:, comment_params:) }
+
+  let(:user)           { create(:user) }
+  let(:book)           { create(:book) }
+  let(:comment)        { Faker::JapaneseMedia::OnePiece.quote }
+  let(:comment_params) { { content: comment, user: } }
+
+  it 'create new comment' do
```

Comment:
`it 'creates new comment' do`

And please add negative scenario when comment is not created